docs/README.md

```diff
@@ -1,11 +1,15 @@
 # Introduction
 
-math.gl is a JavaScript class library for 3D math. It provides the traditional 3D classes (vectors, matrices etc).
+math.gl is a JavaScript class library for 3D and geospatial math. It provides a set of "traditional" 3D classes for vectors, matrices etc, as well as a set of optional modules that address different domains.
 
 In spite of the name, math.gl has no hard WebGL dependencies and is usable in any JavaScript applications. That said, math.gl is designed to be optimized for WebGL/3D applications.
 
+## Available Modules
 
-## Features
+- `math.gl` - basic math classes (vectors, matrices etc).
+- `@math.gl/geospatial` - support for geospatial math, primarily WGS84 and Web Mercator.
```

Comment:
Are you planning to port `viewport-mercator-project` to `@math.gl/geospatial`?

Reply:
> Are you planning to port viewport-mercator-project to @math.gl/geospatial?

Adding support for geospatial projections here is a great idea. Except there is one part of that module which is focused on creating a mapbox compatible projection matrix. I don't think that is totally appropriate for a general purpose geospatial math library.